### DISCUSSION TITLE
Does your Blender2MSFS work with the newest version of Blender 3.4 or 3.3?

### DISCUSSION THREAD

**Discussion by lyonheart001** [tml1024/Blender2MSFS2#16] [Q&A]
Hello TML, 

Just curious if your Blender2MSFS works with Blender 3.3 and up?  On 3.3, the COMP map was changed messing up the exports with the RonH export system.

Bill / lionheart creations

**Comment by Pyscen312** [comment]
No.... the Blender2MSFS only works with 2.83 or 2.93.  The Asobo exporter works with Blender v3.1
